src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -544,19 +545,35 @@ void regclass_Core(py::module m) {
            const py::object& model_stream,
            const std::string& device_name,
            const std::map<std::string, py::object>& properties) {
-            auto _properties = Common::utils::properties_to_any_map(properties);
+            const auto _properties = Common::utils::properties_to_any_map(properties);
             if (!(py::isinstance(model_stream, pybind11::module::import("io").attr("BytesIO")))) {
                 throw py::type_error("CompiledModel.import_model(model_stream) incompatible function argument: "
                                      "`model_stream` must be an io.BytesIO object but " +
                                      (std::string)(py::repr(model_stream)) + "` provided");
             }
             model_stream.attr("seek")(0);  // Always rewind stream!
-            std::stringstream _stream;
-            _stream << model_stream
-                           .attr("read")()  // alternative: model_stream.attr("get_value")()
-                           .cast<std::string>();
+            const std::string filename = "model_stream.txt";
+            std::fstream _stream(filename, std::ios::out | std::ios::binary);
```

Comment:
What about to put the code underneath under
if (_stream.is_open()) {
// use code from underneath
} else {
  OPENVINO_THROW("Failed to open temporary file for model stream");
}

Reply:
And the same for _fstream

Reply:
Done.